src/Symfony/Component/Security/Core/Authorization/Voter/RoleVoter.php

```diff
@@ -41,13 +41,19 @@ public function vote(TokenInterface $token, $subject, array $attributes)
         $roles = $this->extractRoles($token);
 
         foreach ($attributes as $attribute) {
+            if (is_array($attribute)) {
+                continue;
+            }
+            if (is_object($attribute) && !method_exists($attribute, '__toString')) {
+                continue;
+            }
```

Comment:
Statements can be removed. Imo we should not silently ignore invalid values by design.